lib/jsdom/living/nodes/HTMLSelectElement-impl.js

```diff
@@ -90,6 +90,12 @@ class HTMLSelectElementImpl extends HTMLElementImpl {
     return this.options.length;
   }
 
+  get selectedOptions() {
+    return createHTMLCollection(this, () => domSymbolTree.treeToArray(this, { filter(node) {
```

Comment:
Does `createHTMLCollection` ensure that `[SameObject]` works? I think we might have to cache this object in a private var @domenic?